spring-cloud-gcp-autoconfigure/src/main/java/com/google/cloud/spring/autoconfigure/trace/StackdriverTraceAutoConfiguration.java

```diff
@@ -220,8 +226,23 @@ public Sender stackdriverSender(GcpTraceProperties traceProperties,
 	@Bean
 	@ConditionalOnMissingBean
 	public BaggagePropagation.FactoryBuilder baggagePropagationFactoryBuilder() {
-		return BaggagePropagation.newFactoryBuilder(StackdriverTracePropagation.newFactory(
-				B3Propagation.newFactoryBuilder().injectFormat(B3Propagation.Format.MULTI).build()));
+		Propagation.Factory primary = B3Propagation.newFactoryBuilder().injectFormat(B3Propagation.Format.MULTI).build();
```

Comment:
Why did you separate this into two statements?

Reply:
Because it wasn't clear without parentheses counting what was doing what.

Reply:
OK :)